src/ts/mapboxMap.ts

```diff
@@ -200,13 +200,15 @@ module powerbi.extensibility.visual {
 
             //check for GL Support
             // if (!mapboxgl.supported()) {
-            //     this.errorDiv.innerHTML = Templates.noGlSupport
+            //     const text = document.createTextNode(Templates.noGlSupport);
+            //     this.errorDiv..appendChild(text);
             //     return false;
             // }
 
             // Check for Access Token
             if (!this.settings.api.accessToken) {
-                this.errorDiv.innerHTML = Templates.MissingToken;
+                const text = document.createTextNode(Templates.MissingToken);
+                this.errorDiv.appendChild(text);
```

Comment:
Is it possible for this function to be called multiple times and append errors to the error div multiple times?

If so, since it looks like this is the only place we intend to set the content of the div, we should probably remove all of its existing children first:

```
const element = this.errorDiv;
while (element.hasChildNodes()) { element.removeChild(element.firstChild) }
```